### PR TITLE
Added warc header to the template for showing extra data

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Made it so it opens like "all fields"
Made so only one of them is open - and if the other is open when you click, we close it.
Renamed show raw data after request from PO
Did some minor styling.

@thomasegense added for functionality and design approval.
@jorntx added for code review.

### CODE REVIEW

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jorntx) [inline src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue:154]: :))

**Review by thomasegense — CHANGES_REQUESTED** [review]
Warc header information calls wrong url
http://localhost:8080/services/warc/header/?source_file_path=/netarkiv/0229/filedir/330943-325-20200317200216672-00002-kb-prod-har-004.kb.dk.warc.gz&offset=61896515

It is missing /solrwayback (again!)

Can you switch positions for View data and Warc header? The warc header will not be used very much

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jorntx) [inline src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue:155]: :))

**Review by jorntx — COMMENTED** [review]
Fine job - pretty straight forward coding following that path of the _allData_ flow. I only found a few of your boolean shenanigans:)

About the visuals:
- This feature now has the same weight as _show all data_ I got the impression from PO @thomasegense that he wanted a more discrete approach as this is not a much used feature.
- If we keep it this way I miss some kind of indication as to wether the all data "tab" is open or it is the warc header "tab" that it is active. Could be as simple as some visual marking on the toggle button.

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jesperlauridsen) [reply src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue]: Arrrgh!

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jesperlauridsen) [reply src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue:153]: Aarrrrrgh!

**Review by jorntx — APPROVED** [review]
Approved!

**Comment by jorntx** [comment]
Needs sign-off from PO @thomasegense  before I merge

**Review by thomasegense — APPROVED** [review]
OK

**Pull request merged**
